src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,24 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
 
+    public static void main(String[] args) {
+        ManufacturerDao manufacturerDao = (ManufacturerDao) injector
+                .getInstance(ManufacturerDao.class);
+        Manufacturer manufacturer1 = new Manufacturer(1L, "FORD", "USA");
+        Manufacturer manufacturer2 = new Manufacturer(2L, "NISSAN", "JAPAN");
```

Comment:
```suggestion
        Manufacturer fordManufacturer = new Manufacturer("FORD", "USA");
        Manufacturer nissanManufacturer = new Manufacturer("NISSAN", "JAPAN");
```

it's bad practice to use numbers in variable names. And we should not generate id manually, it should be generated by db